typespec-tests/src/main/java/com/cadl/armresourceprovider/models/ChildExtensionResource.java

```diff
@@ -130,7 +130,7 @@ interface WithProperties {
     /**
      * The template for ChildExtensionResource update.
      */
-    interface Update extends UpdateStages.WithProperties {
+    interface Update {
```

Comment:
I need to check why this change.

Reply:
Finally find the cause:
https://github.com/Azure/autorest.java/blob/9b59bd39524bd50b79d04458b464d5d4a241680b/fluentgen/src/main/java/com/azure/autorest/fluent/mapper/ResourceParser.java#L228-L233

We didn't find `PATCH` operation, and fallback to `PUT`, hence the `properties` be present and of type `ChildExtensionResourceInner`.
Reason we didn't find `PATCH` is because its request body is of type `Object`, not ClientModel. We only count request body of type ClientModel be resource `create` or `update` methods. I doubt there are such cases in Swagger brownfield since I can't imagine `PATCH` request body be {"type": "object"}..

In TypeSpec however, the generated [`ResourceUpdateModel`(xxUpdate)](https://github.com/Azure/typespec-azure/blob/f8ee0803166022ec66cc982222d1090aab10cacb/packages/typespec-azure-resource-manager/lib/foundations/arm.foundations.tsp#L114) will [omit](https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/src/private.decorators.ts#L36-L42) `properties` property if it's empty, making `xxUpdate` has no updatable properties.